doc/userguide/ug2html.py

```diff
@@ -155,21 +157,23 @@ def create_userguide():
     os.unlink(version_file)
     os.unlink(install_file)
     ugpath = os.path.abspath(arguments[-1])
-    print ugpath
+    print(ugpath)
     return ugpath, version
 
 
 def _update_version():
     version = _get_version()
-    print 'Version:', version
+    print('Version: %s' % version)
```

Comment:
Using `print('Version', version)` would work fine on Python 3 as well. And we don't need to care about Python 2. Alternatively this script could be updated to Python 3.6 making it possible to use the awesome f-strings.